### PR TITLE
Added OpenCollective banner to the website

### PR DESCRIPTION
**Pull request by xdamman**
...

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -172,6 +172,11 @@ img {
   max-width: 100%;
 }
 
+img[src*="opencollective.com/mochajs"], img[src*=badges] {
```
Comment (dasilvacontin) [inline css/style.css:175]: Oh, thanks for the fix regarding the badges' shadow!

File: index.md [hunk 2]
```diff
@@ -7,6 +7,8 @@ title: "Mocha - the fun, simple, flexible JavaScript test framework"
 Mocha is a feature-rich JavaScript test framework running on [Node.js](http://nodejs.org) and in the browser, making asynchronous testing simple and fun. Mocha tests run serially, allowing for flexible and accurate reporting, while mapping uncaught exceptions to the correct test cases. Hosted on [GitHub](https://github.com/mochajs/mocha).
 
 [![Gitter](//badges.gitter.im/Join%20Chat.svg)](https://gitter.im/mochajs/mocha)
+[![OpenCollective](https://opencollective.com/mochajs/backers/badge.svg)](#backers)
+[![OpenCollective](https://opencollective.com/mochajs/sponsors/badge.svg)](#sponsors)
```
Comment (dasilvacontin) [inline index.md:11]: Alt tag would be better as `OpenCollective backers` and `OpenCollective sponsors`. Might commit that myself.

File: index.md [hunk 3]
```diff
@@ -1031,3 +1035,77 @@ $ make test REPORTER=list
...
+<a href="https://opencollective.com/mochajs/backer/25/website" target="_blank"><img src="https://opencollective.com/mochajs/backer/25/avatar"></a>
+<a href="https://opencollective.com/mochajs/backer/26/website" target="_blank"><img src="https://opencollective.com/mochajs/backer/26/avatar"></a>
+<a href="https://opencollective.com/mochajs/backer/27/website" target="_blank"><img src="https://opencollective.com/mochajs/backer/27/avatar"></a>
+<a href="https://opencollective.com/mochajs/backer/28/website" target="_blank"><img src="https://opencollective.com/mochajs/backer/28/avatar"></a>
+<a href="https://opencollective.com/mochajs/backer/29/website" target="_blank"><img src="https://opencollective.com/mochajs/backer/29/avatar"></a>
+
+
+## Sponsors
+
+Is your company using Mocha? Ask your manager to support us. Your company logo will show up here and on our [Github page](https://github.com/mochajs/mocha#sponsors). [[Become a sponsor](https://opencollective.com/mochajs#sponsor)]
```
Comment (dasilvacontin) [inline index.md:1079]: Github -> GitHub

**Comment by dasilvacontin** [comment]
Cool! I was also gonna add the badges, nice. :)

Currently it looks good. Maybe when we have quite a few backers (making the section very long) would be better to put the sponsors first (which would be one or two lines of images) and then the wall of backers. But looks good as is.

I think I'm gonna merge and polish a few things, add the new logo, etc.

**Comment by dasilvacontin** [comment]
Hmm, gonna try to retinitify / add svg around.

**Comment by dasilvacontin** [comment]
Okay, I believe .svg is not supported yet. Will merge this and will iterate once support for .svg is rolled out.

**Comment by dasilvacontin** [comment]
What I _don't_ like is that backers/sponsors are at the bottom. That's not very good exposure imo.

**Comment by dasilvacontin** [comment]
I locally moved them to the top and I think it's much better. You get to see them when you scroll for the feature list or whatever you are looking for.

If we feel like it's too large we can iterate and make two columns or something.

**Comment by dasilvacontin** [comment]
Gonna merge and iterate. Thanks @xdamman!

**Comment by xdamman** [comment]
Yes, the retina version will be done on our side, you don't need to change anything, that was the whole point of https://github.com/OpenCollective/OpenCollective/issues/44 :-)

**Comment by dasilvacontin** [comment]
Awesome @xdamman :)

**Pull request merged**